### ISSUE TITLE
Unable to connect to host / server - MultiplayerServer

### ISSUE THREAD

**Issue by LetsRepaintIt** [OpenRCT2/OpenRCT2#1935]
Hi guys,

First of all; great initiative!
It would be really great to play RCT2 with more than one person!
Thanks for all your effort so far!

--

Next to that; today I tried creating a MP server for RCT2.
I forwarded my ports to 11753 (and checked it in game settings).
My friend tried to connect, but he received the error; Unable to connect to host.

I'm not using some 3th party program like Hamachi cause that sucks.
I just host it directly from my pc.
Does anyone have a clue?

We've tried running with the verbose command, but no succes so far.
We've also tried running without, but that was even less succesful.

Thanks in advance,

Thomas

**Comment by PFCKrutonium** [comment]
Is your friend connecting to the right IP? Make sure they are connecting to the IP shown here:

https://www.google.ca/search?q=What+is+my+ip+address&rlz=1C1MSNA_enCA657CA657&oq=What+is+my+ip+address&aqs=chrome..69i57.3430j0j7&sourceid=chrome&es_sm=93&ie=UTF-8

**Comment by LetsRepaintIt** [comment]
Hi,

Yes, he is.
He also tried connecting to my public ip + portnumber, but none works.
I set the port to 11753 (standard), but as soon as he tries to connect to
my ip, connection gets refused.
I'm certain I forwarded this same port to my pc as well in my router.
I also have an OpenTTD server, but that one is working just fine.

Thanks for the support,
Thomas

2015-09-16 21:36 GMT+02:00 Krutonium <notifications@github.com>:

**Comment by Patrik356b** [comment]
Check that your forwarding the correct protocol(s)

**Comment by LetsRepaintIt** [comment]
Hi,

I'm always forwarding both TCP and UDP.
Op 17 sep. 2015 00:00 schreef "TheExileFox" <notifications@github.com>:

**Comment by IntelOrca** [comment]
Do you have anything that might be blocking OpenRCT2 from listening on that port? For example Windows Firewall?

**Comment by LetsRepaintIt** [comment]
Uhmm well, I got it allowed in Windows Firewall. But I also tried turning
Firewall off, but it didn't make a difference.
Can't it be some gamesettings that have to be changed?

2015-09-17 14:00 GMT+02:00 Ted John <notifications@github.com>:

**Comment by IntelOrca** [comment]
The only related settings is the default port the game listens on which is 11753. You can double check this by looking at ```Documents/OpenRCT2/config.ini```.

Or you can run it via command line and explicitly set the port.
https://github.com/OpenRCT2/OpenRCT2/wiki/Multiplayer

I don't know what else to try, perhaps a different port that you know works (e.g. the one you use for OpenTTD).

Do you have an IPv4 address? (I am not sure if OpenRCT2 evens supports IPv6).

**Comment by LetsRepaintIt** [comment]
Hi,

Yeah, I have tried to change the port.
It doesn't make a difference. You simply can't connect.
I also tried running it through a command line, but that also didn't matter.
At least; I tried running it through a command line, but not one without
the game itself running.
That's the last thing I can try.. But I'll give it a go.
I'll keep you uptodate.

By the way; Yes, i only use IPv4 ;)

Thomas

2015-09-17 14:13 GMT+02:00 Ted John <notifications@github.com>:
